### PR TITLE
Add IP block registration nodes

### PR DESCRIPTION
**Pull request by mikemoritz**
- New forms intended to closely align with RDAP response structure, with inet:whois:contact eventually being merged into inet:whois:ipcontact

### CODE REVIEW

File: synapse/lib/stormwhois.py [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+        elif form == 'ipquery':
+            guid_props = (
+                props.get('time'),
+                props.get('fqdn'),
+                props.get('url'),
+                props.get('ipv4'),
+                props.get('ipv6'),
+            )
+        else:
+            raise s_exc.StormRuntimeError(mesg=f'No guid helpers available for inet:whois:{form}.')
```
Comment (vEpiphyte) [inline synapse/lib/stormwhois.py]: `form` argument needed as well to be a well-behaved and potentially machine parseable exception..

Comment (mikemoritz) [reply synapse/lib/stormwhois.py]: Updated

File: synapse/models/inet.py [hunk 2]
```diff
@@ -2208,6 +2220,166 @@ def getModelDefs(self):
...
+                            'ro': True,
+                            'doc': 'The classification of the registered network (e.g. direct allocation).'
+                        }),
+                        ('links', ('array', {'type': 'inet:url'}), {
+                            'ro': True,
+                            'doc': 'URLs provided with the record.',
+                        }),
+                    )),
+
+                    ('inet:whois:ipcontact', {}, (
```
Comment (invisig0th) [inline synapse/models/inet.py]: This def continues to make me want form inheritence... :D

File: synapse/models/inet.py [hunk 3]
```diff
@@ -2208,6 +2225,168 @@ def getModelDefs(self):
                         }),
                     )),
 
+                    ('inet:whois:ipquery', {}, (
+                        ('time', ('time', {}), {
+                            'ro': True,
```
Comment (invisig0th) [inline synapse/models/inet.py]: Secondary props which aren't derived from the primary property probably shouldn't be marked read-only

Comment (mikemoritz) [reply synapse/models/inet.py]: updated

**Pull request merged**